python/bot/pythonbot_example.py

```diff
@@ -38,12 +38,29 @@ async def main():
     # The URL path to the ledger you would like to connect to
     url = os.getenv('DAML_LEDGER_URL') or "localhost:6865"
 
-    # The party that is running the automation. The party will _only_ be able to see and operate on
-    # contracts that this party has access to via signatory or observer!
-    party = os.getenv('DAML_LEDGER_PARTY') or "party"
+    # The party that is running the automation if the automation was configured to run as a party.
+    # The party will _only_ be able to see and operate on contracts that this party has access to via
+    # signatory or observer!
+    party = Party(os.getenv('DAML_LEDGER_PARTY') or "party")
+
+    # The name of the application for authorization purposes. If running as a User, the 'application_name'
+    # parameter in dazl.connect _must_ be set to this variable.
+    application_name = os.getenv('DAML_LEDGER_APPLICATION_NAME') or "DAZL-Client"
```

Comment:
I plan to make a change to DAZL to make this unnecessary, but currently it's required to make it work.